### ISSUE TITLE
Allow style pass through with marko helper

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#132]
From original PR #102:

# Description
Add style as a pass through 

## Context

Given that marko treats `class` and `style` differently. We should allow the style to also be explicitly defined in the template, this way we don't run into trouble when using the ebay-button inside a nested component.

```
// this is button wrapper file
<ebay-button ...input />
```

```
// this is the user of the  button-wrapper
$ const style = { color: red }
<button-wrapper fluid style=style > some text <button-wrapper />
```

This doesn't add `style` to the `input['*']` object. And if we otherwise did:
```
// this is button wrapper file
<ebay-button style=input.style />
```

What we get is:

![image](https://user-images.githubusercontent.com/3291619/38955274-2e3d99b0-4322-11e8-9963-ad0f981a980e.png)

**Comment by ianmcburnie** [comment]
@DylanPiercey please verify this was fixed in Marko.

**Comment by DylanPiercey** [comment]
The original issue of the style object not being properly converted to css was fixed in Marko however I just tested this out with the `ebay-button` and noticed that the style is not being passed in under the `*` property. Will chat with @mlrawlings when he gets in today to figure out the best way to move forward with this one. My thoughts are that we should just special case style for now but lets leave this open.
